src/CofreeBot/Bot/Behaviors/Calculator.hs

```diff
@@ -32,10 +32,10 @@ parseErrorBot = pureStatelessBot $ \ParseError {..} ->
 
 simplifyCalculatorBot ::
   Monad m =>
-  Bot m s Program (Either CalcError CalcResp) ->
+  Bot m s Statement (CalcError \/ CalcResp) ->
   Bot m s T.Text T.Text
 simplifyCalculatorBot bot =
-  dimap parseProgram indistinct $ parseErrorBot \/ rmap printCalcOutput bot
+  dimap parseStatement indistinct $ parseErrorBot \/ rmap printCalcOutput bot
```

Comment:
This actually used to be incompatible with the type of `calculatorBot`, but was dead code so wasn't discovered. Now that we use it in a test, need to fix the type.